example/test_problem/ELBDM/LSS_Hybrid/README.md

```diff
@@ -0,0 +1,99 @@
+# `configure.py` options
+- Must enable
+   - [[--model=ELBDM | Installation:-Option-List#--model]]
+   - [[--elbdm_scheme=elbdm_hybrid | Installation:-Option-List#--elbdm_scheme]]
+   - [[--gravity | Installation:-Option-List#--gravity]]
+   - [[--comoving | Installation:-Option-List#--comoving]]
+- Must disable
+   - [[--particle | Installation:-Option-List#--particle]]
+- Available options
+   - [[Miscellaneous Options | Installation:-Option-List#miscellaneous-options]]
+
+
+# Default setup
+1. Cosmological constants
+   | Parameter name | value              |
+   |---             |---                 |
+   | ELBDM_MASS     | 2.0e-23            |
+   | A_INIT         | 9.900990099e-3     |
+   | OMEGA_M0       | 0.3158230904284232 |
+   | HUBBLE0        | 0.6732117          |
+
+2. Simulation
+   | Parameter name         | value |
+   |---                     |---    |
+   | BOX_SIZE               | 2.8   |
+   | MAX_LEVEL              | 6     |
+   | OPT__FLAG_SPECTRAL     | 1     |
+   | OPT__FLAG_INTERFERENCE | 1     |
+
+3. Flag tables
+   * `Input__Flag_Rho`:           `example/input/Input__Flag_Rho8`
+   * `Input__Flag_Spectral`:      `example/input/Input__Flag_Spectral`
+   * `Input__Flag_Interference`:  `example/input/Input__Flag_Interference`
+
+
+Note:
+========================================
+1. Cosmological large-scale structure simulations using hybrid scheme
+
+2. Quickstart:
+   1. Download the IC file
+      ```bash
+      sh download_light_halo_ic.sh
+      ```
+      or
+      ```bash
+      sh download_heavy_halo_ic.sh
+      ```
+   2. For spectral interpolation:
+      -  Download spectral interpolation tables
+         ```bash
+         sh download_spectral_interpolation_tables.sh
+         ```
+      -  Set [[OPT__FLU_INT_SCHEME | ]] and [[OPT__REF_FLU_INT_SCHEME | ]] to 8
```

Comment:
Why is there no link for them? `[[OPT__FLU_INT_SCHEME | Runtime-Parameters:-Interpolation#OPT__FLU_INT_SCHEME ]]`?